backend/src/bundles/chat-messages/types/chat-dto.type.ts

```diff
@@ -0,0 +1,11 @@
+import { type UserDetailsModel } from '~/bundles/user-details/user-details.model.js';
+
+type ChatDto = {
+    chatId: string;
+    lastMessageCreatedAt: string;
+    lastMessage: string;
+    sender: UserDetailsModel;
+    receiver: UserDetailsModel;
+};
```

Comment:
Why we describe Dto type in backend, do you send this DTO to frontend ?

Reply:
This is a local type for repository. All the necessary frontend types are described in shared

Reply:
In this case it can't be DTO type

Reply:
It can be ChatListItem or something like that, but not DTO

Reply:
Change it please !